app/modules/video-players/base.js

```diff
@@ -80,7 +80,7 @@ export default class BasePlayer {
 			};
 		}
 
-		$container.css(sanitizedSize);
+		Object.assign(container.style, sanitizedSize);
```

Comment:
did you test it? It might not work as `sanitizedSize` contains integer values, not string with `px` suffix. I'm just guessing...